ui/src/main/java/edu/wpi/grip/ui/codegeneration/CppTMethods.java

```diff
@@ -16,6 +16,16 @@ public String name(String name) {
   }
 
   @Override
+  public String getterName(String name) {
+    return "get" + name(name);
+  }
+
+  @Override
+  public String setterName(String name) {
+    return name(name);
```

Comment:
Should be `"set" + WordUtils.capitalize(name(name))` (see comment for `getterName`)